infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/msghandlers/LogReplicationServer.java

```diff
@@ -144,7 +143,10 @@ private LogReplicationSession getSession(RequestMsg request) {
             session = LogReplicationSession.newBuilder()
                     .setSourceClusterId(getUUID(request.getHeader().getClusterId()).toString())
                     .setSinkClusterId(localClusterId)
-                    .setSubscriber(LogReplicationConfigManager.getDefaultSubscriber())
+                    .setSubscriber(LogReplication.ReplicationSubscriber.newBuilder()
+                            .setClientName(LogReplicationConfigManager.getDEFAULT_CLIENT())
```

Comment:
incorrect naming convention getDEFAULT_CLIENT.

Reply:
Had DEFAULT_CLIENT as a private field and so had a getter..made it public now.